contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,546 @@
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Apps and integrations</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Push and pull data to other destinations</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Open source</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Full access to source code and transparent roadmap</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Analytics features comparison
+
+Both GA4 and PostHog offer analytics, but _what_ they offer is drastically different. 
+
+GA4 has better features for calculating monetization and ROI on Google Ads and other marketing campaigns. This is particularly helpful for e-commerce websites. 
+
+PostHog, on the other hand, focuses on providing insights on product and feature usage. The biggest difference is [group analytics](/docs/product-analytics/group-analytics), which enable you to track metrics at an account or company level – a feature which GA4 lacks. This makes PostHog especially useful for building B2B SaaS products.
+
+<div className="overflow-x-auto -mx-5 px-5 w-screen md:w-auto">
+  <table className="w-full mt-4">
+    <thead>
+      <tr>
+        <td className="w-3/12"></td>
+        <td class="text-center"><strong>GA4</strong></td>
+        <td class="text-center"><strong>PostHog</strong></td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Group analytics</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Track metrics at account or company level</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Monetization analytics</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Track purchase value, LTV, and other revenue metrics</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Marketing analytics</strong></p>
```

Comment:
I think this implies you can't do marketing analytics using PostHog, which isn't true. Maybe "Advertising analytics" would be better?

Reply:
haha exactly what Ian said